library/src/main/java/com/google/maps/android/clustering/ClusterManager.java

```diff
@@ -363,6 +363,17 @@ public void setOnClusterItemInfoWindowClickListener(OnClusterItemInfoWindowClick
      * Called when an individual ClusterItem is clicked.
      */
     public interface OnClusterItemClickListener<T extends ClusterItem> {
+
+        /**
+         * Called when {@code item} is clicked.
+         *
+         * @param item the item clicked
+         *             
+         * @return true if the handler consumed the event (i.e. the default behavior should not
```

Comment:
If you want to match the `GoogleMap` docs exactly, `handler` would be `listener`. But, LGTM.

https://developers.google.com/android/reference/com/google/android/gms/maps/GoogleMap.OnMarkerClickListener